From eccba61a369fbba3a7d97e6037c2e8a33c1d332d Mon Sep 17 00:00:00 2001
From: Zhihao Cheng <chengzhihao1@huawei.com>
Date: Wed, 16 Nov 2022 10:55:33 +0800
Subject: [PATCH 0045/1398] [Huawei] Revert "[Backport] xfs: add trace for read
 and release of regular file"

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187975

--------------------------------

This reverts commit 83d936acc1c8472053c079d87ef47f0f63f115c7.

Signed-off-by: Zhihao Cheng <chengzhihao1@huawei.com>
---
 fs/xfs/xfs_file.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/fs/xfs/xfs_file.c b/fs/xfs/xfs_file.c
index d2451de87006..3016bb1d5475 100644
--- a/fs/xfs/xfs_file.c
+++ b/fs/xfs/xfs_file.c
@@ -29,7 +29,6 @@
 #include <linux/backing-dev.h>
 #include <linux/mman.h>
 #include <linux/fadvise.h>
-#include <trace/events/fs.h>
 
 static const struct vm_operations_struct xfs_file_vm_ops;
 
@@ -300,7 +299,6 @@ xfs_file_buffered_aio_read(
 	ssize_t			ret;
 
 	trace_xfs_file_buffered_read(ip, iov_iter_count(to), iocb->ki_pos);
-	fs_file_read_do_trace(iocb);
 
 	ret = xfs_ilock_iocb(iocb, XFS_IOLOCK_SHARED);
 	if (ret)
@@ -1219,7 +1217,6 @@ xfs_file_release(
 	struct inode	*inode,
 	struct file	*filp)
 {
-	trace_fs_file_release(inode, filp);
 	return xfs_release(XFS_I(inode));
 }
 
-- 
2.17.1

